src/Raven.Client/Http/ServerNode.cs

```diff
@@ -51,7 +51,31 @@ public override int GetHashCode()
 
         public string LastServerVersion { get; private set; }
 
-        internal bool SupportsAtomicClusterWrites { get; set; }
+        public string ServerVersionFromDatabaseTopology
```

Comment:
rename to ServerVersion

Reply:
It has to be name that different then `ServerVersion`

Reply:
why?

Reply:
Changed

Reply:
Why this is not `internal`? Please explain.

Reply:
It shouldnt be private because we use this in here when we create the json (I think technically it can be private and still has the json field here but it would be a little bit confusing):
https://github.com/ravendb/ravendb/blob/e2f8a383e448104c1a48da7772ed6068359ac399/src/Raven.Server/Web/System/DatabasesHandler.cs#L269